src/sushi.sol

```diff
@@ -58,6 +63,7 @@ contract SushiJoin is CropJoin {
 
     MasterChefLike  immutable public masterchef;
     address                   public initialMigrator;
+    address                   public initialRewarder;
```

Comment:
Since these can both be updated, I'd drop the prefix "initial". If you still want to be descriptive, you could prefix them with e.g. "allowed".